Gemfile

```diff
@@ -11,6 +11,9 @@ gem "sprockets-rails"
 # Use sqlite3 as the database for Active Record
 gem "sqlite3", "~> 1.4"
 
+##TODO change for Postgres once ready to production
+# gem 'pg'
+
```

Comment:
I'd say let's not add dead code (commented code) but yes it would be nice to switch to postgres sooner rather than later!